### DISCUSSION TITLE
Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.

### DISCUSSION THREAD

**Discussion by ric980** [remix-run/remix#9461] [General]
Hello,

When importing tailwind.css, I got this error when starting `yarn dev`.

> Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.

I've set `type=module` in my package.json. That doesn't help.

I don't have any CJS modules. When I comment that tailwind import the warning disappears. 

I import tailwind like so `import styles from '~/tailwind.css'` in my root.tsx.

Everything works but this warning triggers my OCD.

How to avoid that warning?

Thanks.

**Comment by kashw2** [comment]
Hi @ric980, would you by chance be running Node 22.2.0?

**Reply by VIEWVIEWVIEW** [reply]
I'm not OP, but I have the same issue. Yes, I'm running Node 22.2.0. 

Is this a bug in node?

Edit: Downgrading to 22.1.0 doesn't fix the warning. 🤷

**Reply by leumasic** [reply]
Downgrading v20.13.1 fixes it for me

**Comment by kashw2** [comment]
Although I'm not certain. I believe this issue is caused by a 'breaking change' to the Workers API and you can find some discussion in [this issue](https://github.com/nodejs/node/issues/53097).

There are some seemingly related issues with Angular as well which are documented in [this issue](https://github.com/angular/angular-cli/issues/27674).

Although I'm not positive, I'd bargain somewhere in impacted stacks `execArgv` is being passed to `new Worker`.

Myself I'm not using Remix however face the same issues on NodeJS 22.2.0 on a fairly simple React, React Router, Zustand, MUI stack. Where this wasn't an issue past a week or so ago on NodeJS 22.0.0.

On one of my older generations in my NixOS installation which still has NodeJS 22.0.0, this isn't an issue.

**Comment by kerihenare** [comment]
As of Node 22.1, a warning is logged whenever requiring a file fails. Prior to this, it would just throw an error.

Tailwind attempts to require the config file. When that fails (eg: because the file requires transforming), it catches and falls back to transforming the config file: https://github.com/tailwindlabs/tailwindcss/blob/3.3/src/lib/load-config.ts#L29

To avoid the error, just use a Tailwind config file which doesn't require transformation.

**Comment by nicksrandall** [comment]
Thank you! Switching my `tailwind.config.ts` file to `tailwind.config.mjs` fixed the issue for me.

**Reply by iaguirre19** [reply]
I had the same issue, and this solution helped me to fix it. Thanks
